src/services/walktroughServices.js

```diff
@@ -0,0 +1,109 @@
+import { create, watch, currentUser, OpenShiftWatchEvents } from './openshiftServices';
+import {
+  cleanUsername,
+  findOpenshiftResource,
+  buildServiceInstanceResourceObj,
+  findOrCreateOpenshiftResource
+} from './middlewareServices';
```

Comment:
Can we move `cleanUsername`, `findOpenshiftResource` and `findOrCreateOpenshiftResource` functions into commons since these can be reused here. I find it weird to be importing from `middlewareServices` here. wdyt?

I think the `buildServiceInstanceResourceObj` can be imported from `serviceInstanceHelpers`.